.travis.yml

```diff
@@ -5,7 +5,8 @@ node_js:
 - '4'
 before_install:
 - npm install -g npm
-script: npm test
+script: COVERAGE=y npm test
+after_script: "npm install coveralls@2.11.8 && cat ./coverage/lcov.info | coveralls"
```

Comment:
I think it would be better to add it to dev dependencies just to keep the configuration of the build all in one place. I see that it's only needed for Travis but things could get confusing if we only install dependencies on Travis and nowhere else.